src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import 'utils/variables';
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 400;
 }
+
+@import 'blocks/card';
+@import 'blocks/stars';
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), all `@import` statements should be at the top of the file, grouped together without any code in between. Consider moving the `body` styles below the imports to maintain a clean and organized structure.